public/js/controller.js

```diff
@@ -57,9 +57,9 @@ function start() {
 
     var mycamvas = new camvas(window.ctx, processFrame);
     initialized = true;
-    stats.showPanel(0);
-    stats.dom.style.left = '10px';
-    stats.dom.style.top = '200px';
+    // stats.showPanel(0);
```

Comment:
why comment and not remove?

Reply:
I thought maybe for debugging purposes but I can just remove it to be cleaner.